WakaTime/WakaTime.swift

```diff
@@ -158,6 +158,7 @@ enum Category: String {
     case building
     case debugging
     case designing
+    case messaging
```

Comment:
This should be renamed to `communicating` which is the category we use for all voice/text/video messaging.

Reply:
perfect! Will fix.